### PR TITLE
Feature/prlc100 111

### PR DESCRIPTION
**Pull request by kumarmadurai**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/c100-rebuild/childaddress/template.njk [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{% extends "common/template.njk" %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
+{% block page_content %}
+<h1 class="govuk-heading-xl govuk-!-margin-bottom-7">{{headingTitle}}</h1>
+<p class="govuk-body-l">{{paragraph1}}</p>
+<p class="govuk-body-m">{{paragraph2}}</p>
+
+{{ govukWarningText({
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/childaddress/template.njk:10]: @kumarmadurai Can you please use warning macro from the framework ?
{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}

File: src/main/steps/c100-rebuild/childaddress/content.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+});
+
+const cy = () => ({
+  headingTitle: 'Where do the children live? - welsh',
+  paragraph1: `Please tell us the postcode of the children 
+              you’re making this application about. - welsh`,
+  paragraph2: `This information will be used to identify which court will handle your application.
+                If the children have a different postcode, enter the one that is most 
+                convenient to most children in the application. - welsh`,
+  warningText: `You should only enter your own postcode if the children live with you at the address,
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/childaddress/content.ts:33]: We can define warning text like below
`warningText: {
    text: 'The information that you give in this section will also be shared with the other people in this application, so they are able to respond to what you have said.',
    iconFallbackText: 'Warning',
  },`

**Review by channa-patil — CHANGES_REQUESTED** [review]

File: src/main/steps/c100-rebuild/childaddress/template.njk (hunk 1 above)
Comment (kumarmadurai) [reply src/main/steps/c100-rebuild/childaddress/template.njk:10]: I used from framework only. please view the file from top

File: src/main/steps/c100-rebuild/childaddress/content.ts (hunk 2 above)
Comment (kumarmadurai) [reply src/main/steps/c100-rebuild/childaddress/content.ts]: Changes done

**Review by channa-patil — APPROVED** [review]

**Review by channa-patil — APPROVED** [review]

**Review by yogendraU — APPROVED** [review]

**Pull request merged**